Sources/PackageDescription/LanguageStandardSettings.swift

```diff
@@ -10,161 +10,160 @@
+    /// The identifier for the C++14 language standard.
     case cxx14 = "c++14"
 
-    /// ISO C++ 2017 with amendments.
+    /// The identifier for the ISO C++ 2017 (with amendments) language standard..
     @available(_PackageDescription, introduced: 5.4)
     case cxx17 = "c++17"
 
-    /// ISO C++ 2017 with amendments.
+    /// The identifier for the C++1z language standard.
     @available(_PackageDescription, introduced: 4, deprecated: 5.4, renamed: "cxx17")
     case cxx1z = "c++1z"
 
-    /// ISO C++ 2020 DIS.
+    /// The identifier for the ISO C++ 2020 language standard.
     @available(_PackageDescription, introduced: 5.4)
     case cxx20 = "c++20"
 
-    /// Working draft for ISO C++ 2023 DIS.
+    /// The identifier for the ISO C++ 2023 draft language standard.
     @available(_PackageDescription, introduced: 5.6)
     case cxx2b = "c++2b"
 
-    /// ISO C++ 1998 with amendments and GNU extensions.
+    /// The identifier for the GNU++98 language standard.
     case gnucxx98 = "gnu++98"
 
-    /// ISO C++ 1998 with amendments and GNU extensions.
+    /// The identifier for the GNU++03 language standard.
     case gnucxx03 = "gnu++03"
 
-    /// ISO C++ 2011 with amendments and GNU extensions.
+    /// The identifier for the GNU++11 language standard.
     case gnucxx11 = "gnu++11"
 
-    /// ISO C++ 2014 with amendments and GNU extensions.
+    /// The identifier for the GNU++14 language standard.
     case gnucxx14 = "gnu++14"
 
-    /// ISO C++ 2017 with amendments and GNU extensions.
+    /// The identifier for the GNU++17 language standard.
     @available(_PackageDescription, introduced: 5.4)
     case gnucxx17 = "gnu++17"
 
-    /// ISO C++ 2017 with amendments and GNU extensions.
+    /// The identifier for the GNU++1z language standard.
     @available(_PackageDescription, introduced: 4, deprecated: 5.4, renamed: "gnucxx17")
     case gnucxx1z = "gnu++1z"
 
-    /// ISO C++ 2020 DIS with GNU extensions.
+    /// The identifier for the CNU++20 language standard.
     @available(_PackageDescription, introduced: 5.4)
     case gnucxx20 = "gnu++20"
 
-    /// Working draft for ISO C++ 2023 DIS with GNU extensions.
+    /// The identifier for the GNU++2b draft language standard.
     @available(_PackageDescription, introduced: 5.6)
     case gnucxx2b = "gnu++2b"
 }
 
-/// The version of the Swift language to use for compiling Swift sources in the package.
+/// The version of the Swift language to use for compiling Swift sources in the
+/// package.
 public enum SwiftVersion {
+    /// The identifier for the Swift 3 language version.
     @available(_PackageDescription, introduced: 4, obsoleted: 5)
     case v3
 
+    /// The identifier for the Swift 4 language version.
     @available(_PackageDescription, introduced: 4)
     case v4
 
+    /// The identifier for the Swift 4.2 language version.
     @available(_PackageDescription, introduced: 4)
     case v4_2
 
+    /// The identifier for the Swift 5 language version.
```

Comment:
In my opinion, it's easier to read if the documentation for each case is just "Swift 3", "Swift 4.2", etc. Most likely, a user reading it already knows that the symbols are identifiers for Swift language versions, so it's probably better to avoid information overload.

Reply:
We do need a sentence fragment ending in a period, as described in the [API Design Guidelines](https://www.swift.org/documentation/api-design-guidelines/) under "Write a Documentation Comment." So I will leave this as is.